Hunting Queries/W3CIISLog/RareClientFileAccess.yaml

```diff
@@ -1,6 +1,8 @@
 id: a787a819-40df-4c9f-a5ae-850d5a2a0cf6
 name: URI requests from single client
 description: |
+  'Query finds connections to server files requested by only one client. Effective when actor uses static operational IP addresses. Threshold adjustable. Larger execution window increases reliability of results.'
```

Comment:
Description looks broken. Can you rephrase it better?

Reply:
Updated description.